### DISCUSSION TITLE
闲聊】我要休息一阵子

### DISCUSSION THREAD

**Discussion by MistEO** [MaaAssistantArknights/MaaAssistantArknights#3506] [General]
其实我很喜欢自己琢磨代码，感觉很有意思，写 maa 的很大一部分初心也是因为这个。

很高兴 maa 能获得大家的喜欢，给大家提供一些便利也让我很开心。但随着 maa 用户量越来越大，功能越来越多，慢慢的 maa 支持某些新功能变成了一件理所当然的事情，新模式还没出，或者刚出，就有很多朋友来问 maa 会不会支持，还有很多朋友干脆说不想自己打了，等着 maa 支持。

不评判这种行为本身，只是想从我自身的角度出发，我真的有点累了，maa 从兴趣变成了一种责任。写了很多废话又删掉了，总而言之就是我要休息一阵子。大家提的 PR 我有时间还是会认真看的，如果有紧急 bug 我也会第一时间来修，但这段时间我自己的提交会暂缓，主要的更新内容就是依赖大家的 PR，还是欢迎多多提交=。=

大概就是这样，再次感谢大家的支持

**Comment by MistEO** [comment]
然后我说的休息其实也不是纯粹的摆烂，而是去研究其他我觉得有意思的项目了。打个广告，欢迎关注 [我的 B 站账号](https://space.bilibili.com/4506341)，最近可能会整一些比较有意思但又挺无聊的活（

**Comment by YASIN-e** [comment]
毕竟是无偿劳动，并不需要愧疚。做的很好了哦
